src/position/position.js

```diff
@@ -467,6 +467,8 @@ angular.module('ui.bootstrap.position', [])
        *     <li>right-top</li>
        *     <li>right-bottom</li>
        *   </ul>
+       *   @param {boolean=} [shiftTop=false]  - Determines if we should shift the top of the
```

Comment:
I wonder if either this should default to true, or if it should be an option at all and we just always want this behavior.

Reply:
Thinking even more on this, I think we should not use a shiftTop parameter. Instead we should auto shift up/down and left/right if 'auto' is provided in the placement.

Reply:
I can get rid of it if the placement of auto has been provided.